src/test/java/com/google/sps/ProjectCenterTest.java

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 Google LLC
```

Comment:
Copyright year should be 2020.
```suggestion
// Copyright 2020 Google LLC
```